src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -531,6 +530,33 @@ protected void prepareSpawnArea(WorldServer world) {
     }
 
     @Override
+    public Optional<World> copyWorld(World originalWorld, String copyName) {
+        try {
+            FileUtils.copyDirectory(getWorldDir(originalWorld.getName()), getWorldDir(copyName));
```

Comment:
If someone changes the world immediately before cloning it, the changes won't be saved to disk. You may want to save the world before doing this.